Gems/ImGui/Code/Source/LYImGuiUtils/HistogramGroup.cpp

```diff
@@ -77,6 +77,17 @@ namespace ImGui::LYImGuiUtils
             }
         }
     }
+
+    ImGui::LYImGuiUtils::HistogramContainer* HistogramGroup::FindContainerByName(const char* name)
+    {
+        const auto iterator = m_histogramIndexByName.find(name);
+        if (iterator != m_histogramIndexByName.end())
```

Comment:
nit: The scope of this could be reduced to be

```

if (const auto iterator = m_histogramIndexByName.find(name);
    iterator != m_histogramIndexByName.end())
```

Reply:
I find that less clean and the function scope is super small, leaving the current way.